### ISSUE TITLE
Unable to type after deleting text

### ISSUE THREAD

**Issue by peterweck** [quilljs/quill#1254]
Type some text in the editor, then go go back to the beginning of the text and use the "Delete" key to delete all the text. You are no longer able to type any text until you add a return (which you do not see displayed). After adding a return, the new text you type is displayed.

This is only happening on Edge and IE 11, not on Chrome.

Also, oddly enough, this **does not happen on the quill home page** (http://quilljs.com/), but does seem to happen anywhere else.

**Steps for Reproduction**

1. Visit: http://quilljs.com/playground/
2. Click into the editor box where it says "Compose an epic..."
3. Type the word "test"
4. Use the arrow keys to move the start of the word
5. Press the "Delete" key 4 times to delete all the text
6. You are now unable to type without hitting return first

**Expected behavior**:

You should be able to type just as you did when you first entered the editor.

**Actual behavior**:

You are not able to type any new text. 

**Platforms**:

Windows 10 Home, Version 1607 (OS Build 14393.576)
Microsoft Edge 38.14393.0.0 and IE 11 (Version: 11.576.14393.0)

**Version**:

1.1.9

**Comment by jman123** [comment]
I see this same issue on http://quilljs.com/playground/  when using Firefox/Windows 7 and  IE 11/Windows 7.

And this bug happens whether you use the "Delete" key or the "Backspace" key.

**Comment by EladBet** [comment]
Here too, Firefox with windows 10

**Comment by EladBet** [comment]
Didn't fix it on FF nor IE.
You can see the same behavior on [Quill Plyground](http://quilljs.com/playground/#quill-playground)

**Comment by jman123** [comment]
Hi Jason - Has Quill Playground been updated with your latest fix?

On Quill Playground, I'm seeing the same bug on FF/Win 7 and IE11/Win 7. Thanks.

**Comment by jhchen** [comment]
No it has no. You can always check the version by logging Quill.version.

**Comment by EladBet** [comment]
Great thanks

**Comment by peterweck** [comment]
Just tested with a build from the develop branch and is now working. Thanks for the fix!

**Comment by jman123** [comment]
Thanks, Jason. 
Looking forward to your next release that includes this fix.

**Comment by jhchen** [comment]
1.2.0 is now released with the fix hopefully all is well this time. For the record, the reason the quilljs.com homepage did not exhibit this bug is because it included the syntax module which incidentally added additional selection save/restore logic.

**Comment by jbuhawe** [comment]
Hello! 

I would like to ask if this is a bug? I used https://quilljs.com/playground/#custom-fonts in IE 11. The texts do not change into selected font type. Then I tried deleting the whole texts. Change font type first to Mirza. I typed "hello", used left arrow key until I reached the first character then started deleting it. On the first try it was ok. Then I typed again, moved arrow key to the left. When I started hitting delete key, the cursor will jump to the rightmost side. 

Thank you so much!
